pkgs/data/fonts/cabin/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "cabin-1.005";
+
+  src = fetchFromGitHub {
+    owner = "impallari";
+    repo = "Cabin";
+    rev = "982839c790e9dc57c343972aa34c51ed3b3677fd";
+    name = "${name}-src";
+    sha256 = "16v7spviphvdh2rrr8klv11lc9hxphg12ddf0qs7xdx801ri0ppn";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/opentype
+    mkdir -p $out/share/doc/${name}
+    cp -v "fonts/OTF/"*.otf $out/share/fonts/opentype/
+    cp -v README.md FONTLOG.txt $out/share/doc/${name}
+  '';
+
+  outputHashAlgo = "sha256";
+  outputHashMode = "recursive";
+  outputHash = "1ax5c2iab48qsk9zn3gjvqaib2lnlm25f1wr0aysf5ngw0y0jkrd";
```

Comment:
I had a look at where `outputHash` is used today:
```
rycee@beta:~/devel/nixpkgs$ ag -l outputHash
pkgs/applications/networking/browsers/chromium/update.nix
pkgs/applications/networking/cluster/chronos/chronos-deps.nix
pkgs/applications/networking/cluster/mesos/mesos-deps.nix
pkgs/applications/office/libreoffice/default.nix
pkgs/applications/office/libreoffice/still.nix
pkgs/applications/virtualization/virtualbox/default.nix
pkgs/build-support/fetchsvn/default.nix
pkgs/build-support/fetchurl/builder.sh
pkgs/build-support/fetchurl/default.nix
pkgs/build-support/fetchadc/default.nix
pkgs/build-support/fetchbower/default.nix
pkgs/build-support/fetchbzr/default.nix
pkgs/build-support/fetchcvs/default.nix
pkgs/build-support/fetchdarcs/default.nix
pkgs/build-support/fetchegg/default.nix
pkgs/build-support/fetchgit/default.nix
pkgs/build-support/fetchhg/default.nix
pkgs/build-support/fetchmtn/default.nix
pkgs/build-support/fetchsvnssh/default.nix
pkgs/build-support/rust/fetchcargo.nix
pkgs/build-support/docker/pull.nix
pkgs/build-support/fetchfossil/default.nix
pkgs/build-support/trivial-builders.nix
pkgs/data/fonts/google-fonts/default.nix
pkgs/stdenv/adapters.nix
pkgs/tools/typesetting/tex/texlive/default.nix
pkgs/desktops/kde-4.14/kde-base-artwork.nix
pkgs/desktops/kde-4.14/kde-wallpapers.nix
pkgs/desktops/kde-4.14/oxygen-icons.nix
pkgs/games/factorio/fetch.nix
pkgs/servers/xmpp/ejabberd/default.nix
maintainers/scripts/find-tarballs.nix
```
and it seems primarily used to prevent unnecessary mass-rebuilds (e.g., for the `fetchXYZ` functions) or to prevent rebuilding of very large reproducible packages when, e.g., `stdenv` changes.

These packages are indeed reproducible so `outputHash` is quite possible to use but it doesn't quite fit into the usage elsewhere in Nixpkgs. Perhaps this is due to the extra maintenance needed when the package is updated? I'll ask in the IRC channel :-)